e2e/ChannelApeClient.spec.ts

```diff
@@ -739,35 +739,6 @@ describe('ChannelApe Client', () => {
         });
       });
     });
-
-    describe('And valid user business permissions get request', () => {
-      context('When getting user business permissions', () => {
```

Comment:
@mb-ca can't you just add this back in with a different user Id?  You can even do it with the busienss owner: d87e6d12-d7c2-47f7-a7c6-a16b6fcb82f1 ?